fineract-provider/src/main/java/org/apache/fineract/portfolio/note/domain/NoteRepository.java

```diff
@@ -41,5 +41,6 @@
 
     List<Note> findBySavingsAccountId(Long savingAccountId);
 
+    Note findByLoanTransactionId(Long loanTransactionId); 
```

Comment:
By data design there can be multiple notes for a single transaction id.... this API cannot return single Note